shared/templates/kernel_module_disabled/oval.template

```diff
@@ -22,94 +17,43 @@
     <ind:object object_ref="obj_kernmod_{{{ KERNMODULE }}}_disabled" />
   </ind:textfilecontent54_test>
 
-  <ind:textfilecontent54_test id="test_kernmod_{{{ KERNMODULE }}}_modprobeconf" version="1" check="all"
-  comment="kernel module {{{ KERNMODULE }}} disabled in /etc/modprobe.conf">
-    <ind:object object_ref="obj_kernmod_{{{ KERNMODULE }}}_modprobeconf" />
-  </ind:textfilecontent54_test>
-
-  <ind:textfilecontent54_test id="test_kernmod_{{{ KERNMODULE }}}_etcmodules-load" version="1" check="all"
-  comment="kernel module {{{ KERNMODULE }}} disabled in /etc/modules-load.d">
-    <ind:object object_ref="obj_kernmod_{{{ KERNMODULE }}}_etcmodules-load" />
-  </ind:textfilecontent54_test>
-
-  <ind:textfilecontent54_test id="test_kernmod_{{{ KERNMODULE }}}_runmodules-load" version="1" check="all"
-  comment="kernel module {{{ KERNMODULE }}} disabled in /run/modules-load.d">
-    <ind:object object_ref="obj_kernmod_{{{ KERNMODULE }}}_runmodules-load" />
-  </ind:textfilecontent54_test>
-
-  <ind:textfilecontent54_test id="test_kernmod_{{{ KERNMODULE }}}_libmodules-load" version="1" check="all"
-  comment="kernel module {{{ KERNMODULE }}} disabled in /usr/lib/modules-load.d">
-    <ind:object object_ref="obj_kernmod_{{{ KERNMODULE }}}_libmodules-load" />
-  </ind:textfilecontent54_test>
-
-  <ind:textfilecontent54_test id="test_kernmod_{{{ KERNMODULE }}}_runmodprobed" version="1" check="all"
-  comment="kernel module {{{ KERNMODULE }}} disabled in /run/modprobe.d">
-    <ind:object object_ref="obj_kernmod_{{{ KERNMODULE }}}_runmodprobed" />
-  </ind:textfilecontent54_test>
-
-  <ind:textfilecontent54_test id="test_kernmod_{{{ KERNMODULE }}}_libmodprobed" version="1" check="all"
-  comment="kernel module {{{ KERNMODULE }}} disabled in /usr/lib/modprobe.d">
-    <ind:object object_ref="obj_kernmod_{{{ KERNMODULE }}}_libmodprobed" />
-  </ind:textfilecontent54_test>
-
   <ind:textfilecontent54_object id="obj_kernmod_{{{ KERNMODULE }}}_disabled"
   version="1" comment="kernel module {{{ KERNMODULE }}} disabled">
     {{% if product in ["sle12", "sle15"] %}}
```

Comment:
The behavior for SLE12 and SLE15 is different from before, now it only looks for `/etc/modprobe.d/50-blacklist.conf` while before it had the other criteria included:

```
      <criterion test_ref="test_kernmod_{{{ KERNMODULE }}}_etcmodules-load" comment="kernel module {{{ KERNMODULE }}} disabled in /etc/modules-load.d" />
      <criterion test_ref="test_kernmod_{{{ KERNMODULE }}}_runmodules-load" comment="kernel module {{{ KERNMODULE }}} disabled in /run/modules-load.d" />
      <criterion test_ref="test_kernmod_{{{ KERNMODULE }}}_libmodules-load" comment="kernel module {{{ KERNMODULE }}} disabled in /usr/lib/modules-load.d" />
      <criterion test_ref="test_kernmod_{{{ KERNMODULE }}}_runmodprobed" comment="kernel module {{{ KERNMODULE }}} disabled in /run/modprobe.d" />
      <criterion test_ref="test_kernmod_{{{ KERNMODULE }}}_libmodprobed" comment="kernel module {{{ KERNMODULE }}} disabled in /usr/lib/modprobe.d" />
```

That's why they want to follow the STIG precisely

```
Verify the SUSE operating system does not automount USB mass storage devices when connected to the host.

Check that "usb-storage" is blacklisted in the "/etc/modprobe.d/50-blacklist.conf" file with the following command:

> grep usb-storage /etc/modprobe.d/50-blacklist.conf
blacklist usb-storage

If nothing is output from the command, this is a finding.
```

Their original intention was probably how this PR changes things. But the behavior is changing, I'm not sure if that will be a problem. In my opinion, it should be the same check for SLE as well.

Reply:
@teacup-on-rockingchair @brett060102 Hi, are you ok with the change to this template?

Reply:
I have added 6a1b247da1bbb7349d77dda8f25a13dcef760c28 to keep the template behavior in SLE the same.

Reply:
Well, looking at the remediation I don't think the checks I had removed would matter.
The check looks for `Install {{{ KERNMODULE }}}` and the remediation adds `blacklist {{{ KERNMODULE }}}`.

Reply:
I have separated the check for `/etc/modprobe.d/50-blacklist.conf` into its own test.
And the template for SLE will still be checking for same things (except for a minor change, it will also check for `install {{{ KERNMODULE }}}` in `/etc/modprobe.d/*.conf`, should not be a big deal though, I guess).

Reply:
I plan to extend the check for `blacklist` for RHEL in a subsequent PR